lib/version_utils.pm

```diff
@@ -236,3 +239,7 @@ sub is_system_upgrading {
     # If PATCH=1, make sure patch action is finished
     return is_upgrade && (!get_var('PATCH') || (get_var('PATCH') && get_var('SYSTEM_PATCHED')));
 }
+
+sub is_pre_15 {
+    return (is_sle && !sle_version_at_least('15')) || (is_leap && !leap_version_at_least('15.0')) || !is_tumbleweed;
```

Comment:
Logic here is wrong, it returns true for all non-tumbleweed products because of:
```
|| !is_tumbleweed
```

Reply:
I just realized this and have already fixed it locally